### DISCUSSION TITLE
Interest for a vertex coloring algorithm

### DISCUSSION THREAD

**Discussion by AnarchistHoneybun** [petgraph/petgraph#707] [Ideas]
I've been playing around with implementing this paper I found, which describes a wave function collapse coloring algo for graph vertices. Would something like that be of interest to the project?

edit: typo

**Comment by starovoid** [comment]
Hi, that sounds very interesting, please tell how to find this paper.

**Comment by AnarchistHoneybun** [comment]
[This is it](https://arxiv.org/pdf/2108.09329)

Like the dijkstra algo, something like a hashmap that maps each node to some color-id would be good I think?
